package.json

```diff
@@ -72,6 +73,7 @@
     "react-dom": "^16.8.6",
     "react-ga4": "^1.4.1",
     "react-helmet": "^5.2.0",
+    "react-is": "^18.2.0",
```

Comment:
do we need to explicitly add this?

Reply:
iirc the checks were failing without it, or i couldn't start the portal without it, but i'll try again

Reply:
after removing them, i can't start portal locally, i'm getting `Module not found` errors